api/v0/service/handler_test.go

```diff
@@ -456,6 +468,77 @@ func TestGetCodeEmptyOK(t *testing.T) {
 	}, res)
 }
 
+func TestGetExpiryEmptyAddress(t *testing.T) {
+	ctx := context.WithValue(Context, auth.AAD{}, "aad")
+	ctx = context.WithValue(ctx, auth.Session{}, "sessionKey")
+
+	handler := createServiceHandler()
+
+	handler.client.(*MockClient).On("GetExpiry",
+		mock.Anything,
+		backend.GetExpiryRequest{
+			Address: "0x00",
+		}).Return(nil, nil)
+
+	_, err := handler.GetExpiry(ctx, &GetExpiryRequest{
+		Address: "",
+	})
+
+	assert.Error(t, err)
+	baserr := err.(errors.Err)
+
+	assert.Equal(t, "address field has not been set", baserr.Cause().Error())
+	assert.Equal(t, errors.ErrInvalidAddress, baserr.ErrorCode())
+}
+
+func TestGetExpiryEmptyErr(t *testing.T) {
+	ctx := context.WithValue(Context, auth.AAD{}, "aad")
+	ctx = context.WithValue(ctx, auth.Session{}, "sessionKey")
+
+	handler := createServiceHandler()
+
+	handler.client.(*MockClient).On("GetExpiry",
+		mock.Anything,
+		backend.GetExpiryRequest{
+			Address: "0x00",
+		}).Return(nil, errors.New(errors.ErrInternalError, stderr.New("made up error")))
+
+	_, err := handler.GetExpiry(ctx, &GetExpiryRequest{
+		Address: "0x00",
+	})
+
+	assert.Error(t, err)
+	baserr := err.(errors.Err)
+
+	assert.Equal(t, "made up error", baserr.Cause().Error())
+	assert.Equal(t, errors.ErrInternalError, baserr.ErrorCode())
+}
+
+func TestGetExpiryEmptyOK(t *testing.T) {
+	ctx := context.WithValue(Context, auth.AAD{}, "aad")
+	ctx = context.WithValue(ctx, auth.Session{}, "sessionKey")
+
+	handler := createServiceHandler()
+
+	handler.client.(*MockClient).On("GetExpiry",
+		mock.Anything,
+		backend.GetExpiryRequest{
+			Address: "0x00",
+		}).Return(backend.GetExpiryResponse{
+		Address:   "0x00",
+		Expiry:    0,
```

Comment:
I would set a value to the `Expiry` field to make sure it is propagated to the external facing `GetExpiryResponse`